src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,121 @@
+<header>
+  <div class="container-fluid">
+    <nav class="navbar fixed-top navbar-expand-lg navbar-light bg-primary">
```

Comment:
I would recommend using BEM naming convention, but whatever works: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/